ui/appui-react/src/appui-react/accudraw/FrameworkAccuDraw.ts

```diff
@@ -307,11 +308,12 @@ export class FrameworkAccuDraw
   }
 
   private fieldValuesChanged(): void {
-    this.onFieldValueChange(ItemField.X_Item);
-    this.onFieldValueChange(ItemField.Y_Item);
-    this.onFieldValueChange(ItemField.Z_Item);
-    this.onFieldValueChange(ItemField.ANGLE_Item);
-    this.onFieldValueChange(ItemField.DIST_Item);
+    // Only change the value when in Dynamic mode. Other mode are "DontUpdate" when the input is locked and "Partial" when the user is typing.
+    (this.getKeyinStatus(ItemField.X_Item) === KeyinStatus.Dynamic) && this.onFieldValueChange(ItemField.X_Item);
```

Comment:
Checking Dynamic isn't doing anything here right? Because we determined it's always Dynamic currentlym, yes? KeyinStatus is also internal and this code will replaced by calling the new "processMotion" method once I open a new core PR...

Reply:
Oh, I thought that we would still need to do the KeyinStatus check. 
Should I remove it then ?

Reply:
Yeah, I'd revert this, it's not doing anything now and onMotion is going to end up like this:

public override onMotion(_ev: BeButtonEvent): void {
  if (UiFramework.isContextMenuOpen) // I don't know what this is, assume we still want to check it. :)
    return;
  this.processMotion();
}

Reply:
Oh, I just checked the implementation of the `processMotion` function you proposed and it does check the KeyinStatus. 
I'll do an update soon with that code.

Reply:
I removed the Dynamic check, this fix will need to be done in another PR with the new core-frontend API.